doc/api/os.md

```diff
@@ -33,205 +43,281 @@ defined are described in [OS Constants][].
+    times: {
+      user: 308260,
+      nice: 0,
+      sys: 55410,
+      idle: 1071129970,
+      irq: 880
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 266450,
-       nice: 1480,
-       sys: 34920,
-       idle: 1072572010,
-       irq: 30 } } ]
+    times: {
+      user: 266450,
+      nice: 1480,
+      sys: 34920,
+      idle: 1072572010,
+      irq: 30
+    }
+  }
+]
 ```
 
-Note that since `nice` values are UNIX centric in Windows the `nice` values of
+*Note*: Because `nice` values are UNIX-specific, on Windows the `nice` values of
 all processors are always 0.
 
 ## os.endianness()
 <!-- YAML
 added: v0.9.4
 -->
 
-Returns the endianness of the CPU. Possible values are `'BE'` for big endian
-or `'LE'` for little endian.
+The `os.endianness()` method returns the endianness of the CPU.
+
+Possible values are:
+
+* `'BE'` for big endian
+* `'LE'` for little endian.
 
 ## os.freemem()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns the amount of free system memory in bytes.
+The `os.freemem()` method returns the amount of free system memory in bytes.
 
 ## os.homedir()
 <!-- YAML
 added: v2.3.0
 -->
 
-Returns the home directory of the current user.
+The `os.homedir()` method returns the home directory of the current user as a
+string.
 
 ## os.hostname()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns the hostname of the operating system.
+The `os.hostname()` method returns the hostname of the operating system as a
+string.
 
 ## os.loadavg()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns an array containing the 1, 5, and 15 minute load averages.
+The `os.loadavg()` method returns an array containing the 1, 5, and 15 minute
+load averages.
 
 The load average is a measure of system activity, calculated by the operating
 system and expressed as a fractional number.  As a rule of thumb, the load
 average should ideally be less than the number of logical CPUs in the system.
 
-The load average is a very UNIX-y concept; there is no real equivalent on
-Windows platforms.  That is why this function always returns `[0, 0, 0]` on
-Windows.
+The load average is a UNIX-specific concept with no real equivalent on
+Windows platforms. On Windows, the return value is always `[0, 0, 0]`.
 
 ## os.networkInterfaces()
 <!-- YAML
 added: v0.6.0
 -->
 
-Get a list of network interfaces:
+The `os.networkInterfaces()` method returns an object listing only network
+interfaces that have been assigned a network address.
+
+Each key on the returned object identifies a network interface. The associated
+value is an array of objects that each describe an assigned network address.
+
+The properties possible on the assigned network address object include:
+
+* `address` {String} The assigned IPv4 or IPv6 address
+* `netmask` {String} The IPv4 or IPv6 network mask
+* `family` {String} Either `IPv4` or `IPv6`
+* `mac` {String} The mac address of the network interface
+* `internal` {boolean}
```

Comment:
This property indicates whether the network interface is a loopback or similar interface, one that cannot be reached outside of the system.